### ISSUE TITLE
Activable Stash/Group

### ISSUE THREAD

**Issue by FabianoLothor** [josh-berry/tab-stash#341]
### Problem Statement

As a user.
I want to be see the open tabs of a single stash/group.
Then I can have my

### Preferred Solution(s)

Add a new configuration to the settings page to change the behavior of the two "open all tabs" buttons (yellow and green).

When enabled, the buttons will first hide all currently open tabs and reopen only the ones that belong to that stash/group.

### Alternative Solution(s)

N/A

### Additional Context

The reason for that is because sometimes you don't want to see all open tabs, only the context that you are working.

<hr>
<em>Readers: If you are also interested in seeing this feature be developed, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the feature request, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
Thanks for the feature request; I'll keep it in mind!

This seems related to, but I think slightly different from:
* #107 
* #42 

#107 is asking to open a group in a new window; this seems to be asking to replace all the tabs in the *current* window with the tabs from the desired group. Whereas #42 would cover the inverse—you have a bunch of tabs open, and you want to *replace* the contents of the group with the open tabs.

**Comment by FabianoLothor** [comment]
Yes @josh-berry, I know that firefox has an option to hide tabs.

Sidebery handles it very well, it worth to take a look to know how does it work.

**Comment by nmeier** [comment]
I think this is the same things as per OP: I'm looking for a way to quickly stash away the current set of tabs and open all tabs from a stash (group). This would help me switch between different phases of work (each with a set of different tabs)
,
work on project 1 > work on project 2 > don't work ... email, twitter, news > back to project 1 > spend 30 mins in email, calendar and approval tools

It seems that each switch currently requires two clicks - stash what I have, open all tabs (green) - repeat.
